signer/src/storage/postgres.rs

```diff
@@ -450,8 +450,8 @@ impl PgStore {
     async fn get_least_txo_height(
         &self,
         chain_tip: &model::BitcoinBlockHash,
-        min_block_height: i64,
-    ) -> Result<Option<i64>, Error> {
+        min_block_height: BitcoinBlockHeight,
+    ) -> Result<Option<BitcoinBlockHeight>, Error> {
         sqlx::query_scalar::<_, i64>(
```

Comment:
I think we can do this:
```suggestion
        sqlx::query_scalar::<_, BitcoinBlockHeight>(
```
Then there is no need to do the conversion afterwards.

Reply:
Whow nice catch! [776797d](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/776797daa99c421255b29467ac9e57e8322bff18)